playbooks/provisioning/openstack/README.md

```diff
@@ -221,6 +221,28 @@ Once it succeeds, you can install openshift by running:
     ansible-playbook openshift-ansible/playbooks/byo/config.yml
 
 
+## Scale Deployment up/down
+
+### Scaling up
+
+1. Make sure that `openshift-ansible-contrib/playbokos/provisioning/openstack/upscaling_run.sh` is executable. If not, run `chmod +x <file>`
+2. Check that `ansible_become: true` is set in `<inventory>/group_vars/OSEv3.yml`
+3. Go to a directory that contains both `inventory` and `openshift-ansible*` directories.
+4. Run the script:
+
+    ./openshift-ansible-contrib/playbooks/upscaling_run.sh -i <path to inventory>
+    [-e nodes=<new number of nodes>] [-h] [-v]
```

Comment:
Github's rendering seems to eat the angle brackets for some reason even though this should be a code block because markdown:

https://github.com/Tlacenka/openshift-ansible-contrib/blob/88460aa65f62e56dca0e04558059f5153e81d9ef/playbooks/provisioning/openstack/README.md

Try wrapping it in blocks of triple backticks like so:

    ```
    ./openshift-ansible-contrib/playbooks/upscaling_run.sh -i <path to inventory> [-e nodes=<new number of nodes>] [-h] [-v]
    ```